.github/workflows/auto-label.yml

```
@@ -0,0 +1,37 @@
+on:
+  issues:
+    types:
+      - edited
+      - opened
+
+jobs:
+  label:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Apply Labels to Issues
+        uses: actions/github-script@v6
```

actionlint (1.7.4)

the runner of "actions/github-script@v6" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)